core/src/main/java/org/elasticsearch/transport/RequestHandlerRegistry.java

```diff
@@ -77,6 +80,10 @@ public boolean isForceExecution() {
         return forceExecution;
     }
 
+    public boolean isCheckSizeLimit() {
```

Comment:
Can we change the name of this (and the corresponding field)? Maybe `canTripCircuitBreaker`?